include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -322,19 +322,18 @@
   // two loops execute in series in a parallel section. ]
   virtual void RunInParallel(std::function<void(unsigned idx)> fn,
                              unsigned n, std::ptrdiff_t block_size) = 0;
-  virtual void StartProfiling()  = 0;
+  virtual void StartProfiling() = 0;
   virtual std::string StopProfiling() = 0;
 };
 
-
 class ThreadPoolParallelSection {
  public:
   // State accessed only by the main thread
   // --------------------------------------
 
   // Tasks successfully submitted to the work queues.  This sets the
   // maximum degree of parallelism that the section will support.
-  std::vector<std::pair<int,unsigned>> tasks;
+  std::vector<std::pair<int, unsigned>> tasks;
```

Comment:
>vector [](http://example.com/codeflow?start=7&length=6)

Shall we switch the container to use inlined?

Reply:
I would do it in a separate PR if required. I tried it and I did not see a perf gain for the problem at hand.

Reply:
I do not think the usage of InlinedVector should be conditional on perf, just like std::string with its small buffer is not. In this PR there are other things to worry about.
The second argument creates more instantiations which we probably do not need, except may be for shape and convolutional paddings.